### ISSUE TITLE
Get rid of the wann2kc.w90_files

### ISSUE THREAD

**Issue by mikibonacci** [mikibonacci/aiida-koopmans#5]
This can be done by using the self.link(Process, wann2kc_calc) and use the dst_path name to understand the attribute.

then, in the helper used to populate the builder, I use a pop to automatically delete the custom attribute.

**Comment by GeigerJ2** [comment]
Note: decorate `self._fetch_linked_files`, not `self.link`.

**Comment by mikibonacci** [comment]
Exactly, so the idea would be to decorate not the link but the _fetch_linked_files, which contains the src_calc attribute.
